Dockerfile

```diff
@@ -1,11 +1,18 @@
-FROM node:10.6.0
+# Stage 1
+FROM node:10.8 as yarnbuild
 
 WORKDIR /data
 COPY . /data/
 
 RUN npm install
 RUN npm run install
```

Comment:
I think both these two lines can be replaced with a single `yarn install` command, see the one that we're using [here](https://github.com/KSF-Media/gitlab-dashboard/blob/master/.gitlab-ci.yml#L18)

Reply:
Great, I will try it out and push a new change then. Thanks.

Reply:
it works indeed fine. So I've pushed a new commit with this change.